docs/news/news-v5.0.x.rst

```diff
@@ -11,254 +11,335 @@ Open MPI version 5.0.0rc12
 .. admonition:: MPIR API has been removed
    :class: warning
 
-   As was announced in summer 2017, Open MPI has removed support of
-   MPIR-based tools beginning with the release of Open MPI v5.0.0.
+   As was announced in the summer of 2017, Open MPI has removed
+   support for MPIR-based tools beginning with the release of Open MPI
+   v5.0.0.
 
-   The new PRRTE based runtime environment supports PMIx-tools API
-   instead of the legacy MPIR API for debugging parallel jobs.
+   Open MPI now uses the `PRRTE <https://github.com/openpmix/prrte>`_
+   runtime environment, which supports the `PMIx <https://pmix.org/>`_
+   tools API |mdash| instead of the legacy MPIR API |mdash| for
+   debugging parallel jobs.
 
-   See https://github.com/hpc/mpir-to-pmix-guide for more
-   information.
+   Users who still need legacy MPIR support should see
+   https://github.com/hpc/mpir-to-pmix-guide for more information.
 
-.. admonition:: Zlib is suggested for better user experience
+.. admonition:: Zlib is suggested for better performance
    :class: note
 
-   PMIx will optionally use `Zlib <https://github.com/madler/zlib>`_
-   to compress large data streams.  This may result in faster startup
-   times and smaller memory footprints (compared to not using
-   compression).  The Open MPI community recommends building Zlib
-   support with PMIx, regardless of whether you are using an
-   externally-installed PMIx or the PMIx that is installed with Open
-   MPI.
-
-.. caution::
-   Open MPI no longer builds 3rd-party packages
-   such as Libevent, HWLOC, PMIx, and PRRTE as MCA components
-   and instead:
-
-   #. Relies on external libraries whenever possible, and
-   #. Builds the 3rd party libraries only if needed, and as independent
-      libraries, rather than linked into the Open MPI core libraries.
-
+   `PMIx <https://pmix.org/>`_ will optionally use `Zlib
+   <https://github.com/madler/zlib>`_ to compress large data streams.
+   This may result in faster startup times and smaller memory
+   footprints (compared to not using compression).
+
+   The Open MPI community recommends building PMIx with Zlib support,
+   regardless of whether you are using an externally-installed PMIx or
+   the bundled PMIx that is included with Open MPI distribution
+   tarballs.
```

Comment:
We probably should add something like: PMIx will automatically build with Zlib support if it finds the Zlib headers and libraries; most systems do not install the Zlib development headers by default.